libs/hash/include/nil/crypto3/hash/detail/poseidon/poseidon_constants.hpp

```diff
@@ -30,7 +30,7 @@ namespace nil {
             namespace detail {
 
                 template<typename PolicyType>
-                class poseidon_constants {
+                struct poseidon_constants {
```

Comment:
Why change this to struct?

Reply:
Because generic template is struct:
https://github.com/NilFoundation/crypto3/blob/master/libs/hash/include/nil/crypto3/hash/detail/poseidon/poseidon_constants_generator.hpp#L22